airflow/contrib/operators/gcs_download_operator.py

```diff
@@ -1,8 +1,9 @@
 import logging
+import os
 
 from airflow.contrib.hooks.gcs_hook import GoogleCloudStorageHook
 from airflow.models import BaseOperator
-from airflow.utils.decorators import apply_defaults
+from airflow.utils import apply_defaults
```

Comment:
I think this should be left as `airflow.utils.decorators`. The `airflow.utils` hack has been deprecated:

```
def apply_defaults(func):
    warnings.warn_explicit(
        """
        You are importing apply_defaults from airflow.utils which
        will be deprecated in a future version.
        Please use :

        from airflow.utils.decorators import apply_defaults
        """,
        category=PendingDeprecationWarning,
        filename=func.__code__.co_filename,
        lineno=func.__code__.co_firstlineno + 1
    )
    return _apply_defaults(func)
```